macos-trust-cert.app/Contents/Info.plist

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple Computer//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+  <key>CFBundleDisplayName</key>
+  <string>macos-trust-cert</string>
```

Comment:
Is this surfaced in any UI somewhere? If yes, probably makes sense to identify it as mitmproxy here, e.g. `mitmproxy certificate installer`. If that's not the case it does not matter obviously. :)